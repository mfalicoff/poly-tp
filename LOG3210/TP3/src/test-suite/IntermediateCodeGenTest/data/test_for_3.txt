Declaration
num a;
num i;
num j;
EndDeclaration

a = 0;

for (i = 3; i <= 5; i = i + 2) {
  a = 0;
}
