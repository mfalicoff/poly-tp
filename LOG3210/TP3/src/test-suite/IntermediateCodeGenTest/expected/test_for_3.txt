a = 0
_L1
i = 3
_L2
if i <= 5 goto _L4
goto _L0
_L4
a = 0
_L3
_t0 = i + 2
i = _t0
goto _L2
_L0
